solutions/1584/01.py

```
@@ -0,0 +1,32 @@
+class Solution:
+    def minCostConnectPoints(self, points: List[List[int]]) -> int:
```

Ruff (0.14.7)

Undefined name `List`

(F821)